packages/lesswrong/components/recommendations/RecommendationsAlgorithmPicker.tsx

```diff
@@ -97,38 +97,34 @@ const RecommendationsAlgorithmPicker = ({ settings, configName, onChange, showAd
     onChange(newSettings);
   }
   return <div className={classes.root}>
-    <span className={classes.settingGroup}>
+    {['frontpage', 'frontpageEA'].includes(configName) && <span className={classes.settingGroup}>
       <span className={classes.setting}>
-        {(configName === "frontpage") &&
-          <SectionFooterCheckbox
-            value={!settings.hideContinueReading}
-            onClick={(ev, checked) => applyChange({ ...settings, hideContinueReading: !settings.hideContinueReading })}
-            label="Continue Reading"
-            tooltip="If you start reading a sequence, the next unread post will appear in Recommendations"
-          />
-        }
+        <SectionFooterCheckbox
+          value={!settings.hideContinueReading}
+          onClick={(ev, checked) => applyChange({ ...settings, hideContinueReading: !settings.hideContinueReading })}
+          label="Continue Reading"
+          tooltip="If you start reading a sequence, the next unread post will appear in Recommendations"
+        />
       </span>
       <span className={classes.setting}>
-        {(configName === "frontpage") && 
-          <SectionFooterCheckbox
-            value={!settings.hideBookmarks}
-            onClick={(ev, checked) => applyChange({ ...settings, hideBookmarks: !settings.hideBookmarks })}
-            label="Bookmarks"
-            tooltip="Posts that you have bookmarked will appear in Recommendations."
-          />
-        }
+        <SectionFooterCheckbox
+          value={!settings.hideBookmarks}
+          onClick={(ev, checked) => applyChange({ ...settings, hideBookmarks: !settings.hideBookmarks })}
+          label="Bookmarks"
+          tooltip="Posts that you have bookmarked will appear in Recommendations."
+        />
       </span>
-    </span>
+    </span>}
 
     {/* disabled except during review */}
-    {/* {(configName === "frontpage") && <div> 
+    {/* {(configName === "frontpage") && <div>
```

Comment:
Assuming this is been left here to reduce diff. If not 💀

Reply:
We're submitting upstream, so it's not that. But LW seems to want to keep this around until the next review. Doesn't seem insane.